themes/edx.org/lms/templates/dashboard.html

```diff
@@ -288,6 +288,22 @@ <h3>${course_dir}</h3>
       </div>
   % endif
 
+  <% account_mfe_url = getattr(settings, 'ACCOUNT_MICROFRONTEND_URL', '') or '' %>
+  % if plugins.get("coaching", {}).get("show_coaching_aside"):
+    <div class="wrapper-coaching">
+        <div class="coaching-signup">
+            <div class="coaching-prompt">
+                ${_("Take advantage of free coaching!")}
+            </div>
+            <div class="coaching-link">
+                <a class="btn-neutral" href="${account_mfe_url}">
+                  ${_("Get Started")}
```

Comment:
Will these strings be automatically translated?

Do we need to do anything to make this accessible?

Reply:
After looking at the previous transifex bot PRs, it looks like the bot handles the extraction of new translations now. Going to merge this as is and keep and eye for the bot's next PR.